settings.gradle

```diff
@@ -1 +1,30 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        maven {
+            name 'eveohInternalGradlePlugins'
+            credentials(PasswordCredentials)
+            url 'https://maven.eveoh.nl/repository/internalgradleplugins/'
+        }
+    }
+    plugins {
+        id 'com.bmuschko.tomcat' version '2.2.5'
+        id 'nl.eveoh.gradle.verifications' version '0.0.1'
+    }
+}
+
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+        maven {
+            name 'eveohBlackboard'
```

Comment:
Betekent dit (en die hierboven) niet dat je meer regels op moet nemen in je lokale `gradle.properties`, en moeten we de wiki pagina daar niet op aanpassen? Daar staat nu 1 voorbeeldje (https://wiki.eveoh.nl/pages/viewpage.action?spaceKey=DEV&title=Internal+Maven+repositories#InternalMavenrepositories-Configuringcredentials) maar het lijkt mij handiger als je als developer gewoon het hele lijstje kan kopiëren en in kan vullen?

Reply:
Ik wilde die wiki ook niet te expliciet maken, omdat het dan sneller outdated info is. Ik heb bijvoorbeeld ook niet die blackboard repo in de lijst opgenomen, omdat hij maar voor één project relevant is.

De developer mag ook wel begrijpen hoe het mechanisme werkt en krijgt ook een heel duidelijke error van Gradle.

Reply:
Ik snap dat deze niet zo relevant is, maar voor de internal gradle plugins is het toch iets wat iedere developer altijd moet hebben. Die staat nu niet in de lijst of de default config, en daarmee wordt het uitchecken van het project een soort trial and error voor de developer.

Het lijkt mij toch handig die wel in een lijstje op te nemen, en ergens neer te zetten dat `eveohInternal` en `eveohInternalGradlePlugins` met name relevant zijn voor de developers.